lua/exporters/object.lua

```diff
@@ -8,32 +8,32 @@
 local GRPC = GRPC
 local coord = coord
 
-local function toLatLonPosition(pos)
-  local lat, lon, alt = coord.LOtoLL(pos)
-  return {
-    lat = lat,
-    lon = lon,
-    alt = alt,
-  }
-end

```

Comment:
This already existed as `GRPC.toLatLonPosition`

Reply:
Whoops, understood